scripts/SREP/hard-prune-image-registry/metadata.yaml

```
@@ -0,0 +1,146 @@
+file: script.sh
+name: hard-prune-image-registry
+shortDescription: Manually prune image registry
+description: Manually prunes orphaned blobs in the image registry; this should be used sparingly
```

```suggestion
description: Hard prunes orphaned blobs in the image registry; this causes the image registry to enter read-only mode during the pruning and should be used only when the customer acknowledges this. https://docs.openshift.com/container-platform/4.13/applications/pruning-objects.html#pruning-hard-pruning-registry_pruning-objects
```

Doesn't have to be this exact wording, but I was thinking we can elaborate on "sparingly" and link to the docs behind this process